src/util/rename_symbol.cpp

```diff
@@ -170,7 +170,7 @@ bool rename_symbolt::rename(typet &dest) const
       }
     }
   }
-  else if(dest.id()==ID_symbol)
+  else if(dest.id() == ID_symbol_type)
```

Comment:
As (now!) per coding standards these should be `can_cast_type<symbol_typet>(dest)`